.cirrus.yml

```diff
@@ -113,8 +113,6 @@ debian_testing_task: &debian_testing
             TEST_DRIVER: "fuse-overlay-whiteout"
         - env:
             TEST_DRIVER: "btrfs"
-        - env:
-            TEST_DRIVER: "zfs"
```

Comment:
Please ref. related commit message in this PR.

Reply:
> ZFS isn't supported in debian, it's always been a community side-load
situation (think EPEL for CentOS).  This is the only containers repo.
that makes use of it in CI.  Worse, since we're testing with SID,
installing (i.e. compiling) the ZFS kernel module is an even more
fragile and failure prone process.  Because of those issues I've
stripped it out of these CI VM images.  I no-longer wish to support
its use in this CI setup.  If somebody else wants to support it, they're
more than welcome and free to do so.